README.md

```diff
@@ -1 +1,26 @@
 # FINDOLOGIC Shopware 6 Plugin Override
+
+This is a basic plugin skeleton which may be used to customize
+[FINDOLOGIC's Shopware 6 plugin](#).
+
+## Requirements
+Note that in order to run this plugin you will need to install
```

Comment:
Change this to:

* [FINDOLOGIC Shopware 6 plugin](https://github.com/findologic/plugin-shopware-6).